internal/services/config/config.go

```diff
@@ -261,6 +264,10 @@ var defaultConfig = Config{
 		},
 		ActiveTasksLimit: 2,
 	},
+	Gitserver: Gitserver{
+		ScanTime:                     24 * time.Hour,
```

Comment:
Lets call it `RepositoryCleanupInterval`. See previous comment: https://github.com/agola-io/agola/pull/284#discussion_r795554221